src/components/Grid/Grid.tsx

```diff
@@ -38,6 +38,25 @@ export const Grid: React.FC<GridProps> = ({
   const [itemShowingDialog, setItemShowingDialog] =
     useState<CommonItemType | null>(null);
 
+  const sortItems = (a: TopicMapItemType, b: TopicMapItemType): number => {
+    // Sort after index first
+    if (a.index && b.index) {
```

Comment:
The index might be `0` which is falsy, however should be sorted before 1, in my opinion.

```suggestion
    if (a.index != null && b.index != null) {
```

Reply:
Yes, thanks!